pkg/osquery/extension.go

```diff
@@ -556,6 +557,13 @@ func (e *Extension) generateConfigsWithReenroll(ctx context.Context, reenroll bo
 
 	config = e.setOsqueryOptions(config, configOptsToSet)
 
+	// If this feature flag is set, we want to use cached data for scheduled queries that might otherwise
+	// time out. We achieve this by updating the config to point to the tables holding the cached data.
+	if e.knapsack.UseCachedDataForScheduledQueries() {
+		// The kolide_windows_updates table is currently the only table we use this feature for.
+		config = strings.ReplaceAll(config, "kolide_windows_updates", "kolide_windows_updates_cached")
```

Comment:
worth remembering that this is now a magic string that will explode if a customer uses it.

Reply:
Maybe worth going through and only updating this string if it's in a pack => query section of the config?

Reply:
Discussed offline, concluded not worth it!